### PR TITLE
Newsletter-223: Translate to Hindi

### PR DESCRIPTION
**Pull request by shaavan**

### CODE REVIEW

**Comment by shaavan** [comment]
Updated from fb6e767 to fef62ec

Changes:

- Fixed a trailing whitespace error.

File: _posts/hi/newsletters/2022-10-26-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+[rubin cpfp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021059.html
+[riard free option]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021067.html
+[zhao no control]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-440
+[towns uncoordinated]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-488
+[sanders doubt]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-490
+[towns uasf]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-492
+[zumsande option]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-493
+[coredev xs]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/
+[p2p encryption]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/2022-10-10-p2p-encryption/
+[news222 bip324]: /en/newsletters/2022/10/19/#bip324-update
```
Comment (bitschmidty) [inline _posts/hi/newsletters/2022-10-26-newsletter.md:239]: /hi/ version available

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Just one correction @shaavan

File: _posts/hi/newsletters/2022-10-26-newsletter.md (hunk 1 above)
Comment (shaavan) [reply _posts/hi/newsletters/2022-10-26-newsletter.md:239]: First, I added the `hi` version of the link here. But for some reason, it was causing the following error.

image

So I made an exception to reverting to the `en` version for this link.

Comment (bitschmidty) [reply _posts/hi/newsletters/2022-10-26-newsletter.md:239]: @shaavan ping on this

Comment (bitschmidty) [reply _posts/hi/newsletters/2022-10-26-newsletter.md]: @shaavan I pushed a change to this link.

**Pull request merged**